controller/test.mustache

```diff
@@ -1,23 +1,37 @@
-"use strict";
+{{=<% %>=}}
```

Comment:
It clashes when trying to do the string interpolation stuff in the template strings.
It doesn't like `\`something ${{{pluralName}}} else\``